docs/sphinx_rsts/intro/using_the_library.rst

```diff
@@ -7,33 +7,19 @@ The source files in the demo directories, particularly those in src/examples/pke
 
 To use OpenFHE, you must
 
-* select a lattice element to be used for your application. We support:
-
- * ``Poly``
-
- * ``NativePoly``
-
- * ``DCRTPoly``
-
 * select a scheme to use
 
-  * ``BFV``
-
   * ``BFVrns``
 
-  * ``BFVrnsB``
-
   * ``BGVrns``
 
-  * ``CKKS``
-
-  * ``Null`` (a test scheme which does not actually encrypt/decrypt anything)
```

Comment:
Did we remove the `Null` scheme? Is there a reason why we did it?

Reply:
yes, we removed it. The previous null scheme was only for BGV and BFV. It was more confusing to keep it.